src/scanner/index.ts

```diff
@@ -5,64 +5,87 @@ import { constructDeleteWorkload, constructDepGraph } from '../transmitter/paylo
 import { IWorkload, ILocalWorkloadLocator } from '../transmitter/types';
 import { IPullableImage } from './images/types';
 
-export = class WorkloadWorker {
-  private readonly name: string;
+export async function processWorkload(
+  workloadMetadata: IWorkload[],
+): Promise<void> {
+  // every workload metadata references the same workload name, grab it from the first one
+  const workloadName = workloadMetadata[0].name;
 
-  constructor(name: string) {
-    this.name = name;
-  }
-
-  public async process(workloadMetadata: IWorkload[]): Promise<void> {
-    const workloadName = this.name;
-    const allImages = workloadMetadata.map((meta) => meta.imageName);
-    logger.info({workloadName, imageCount: allImages.length}, 'queried workloads');
-    const uniqueImages = [...new Set<string>(allImages)];
+  const allImages = workloadMetadata.map((meta) => meta.imageName);
+  logger.info(
```

Comment:
are you sneaking prettify into our code base line by line? 😆
jokes aside, it makes it harder to review what changed and what just switched places